blaze/compute/tests/test_numpy_compute.py

```diff
@@ -18,6 +16,10 @@
 import blaze
 from odo import into
 from datashape import discover, to_numpy, dshape
+try:
+    import numba
+except ImportError:
+    numba = None
```

Comment:
I think the only reason these fail is because I'm using a `numba.TypingError` in a `raises` argument somewhere below. That test could be moved to another file and then you wouldn't need to import numba here. Generally these kinds of imports should be avoided as they hide errors